src/Containers/Microsoft.NET.Build.Containers/KnownStrings.cs

```diff
@@ -49,8 +49,14 @@ public static class Items
 
     public static class ErrorCodes
     {
+        // current version doesn't support containerization
         public static readonly string CONTAINER002 = nameof(CONTAINER002);
+        // containerimagename rename
         public static readonly string CONTAINER003 = nameof(CONTAINER003);
+        // generic http error
+        public static readonly string CONTAINER004 = nameof(CONTAINER004);
+        // don't use the containers package
+        public static readonly string CONTAINER005 = nameof(CONTAINER005);
```

Comment:
🤔 Is there an actual list of what *all* of these errors mean? Might make sense to name them differently. You couldn't use `nameof` then but at least it would be more human understandable.

Reply:
No, there's not - this is mainly used as a kind of 'registry' similar to how we keep a list for the SDK codes and have a comment that says "the last number used was XYZ". The goal would be for each of these to have help docs describing them on learn.ms.com, but we don't yet have that.

Reply:
Oh, I see. So, why was 004 added to this PR but not used? Or did it already exist in the code as a literal but not as a static string?

Reply:
Exactly - it's in the `ContainerHttpError` exception type used in some of the internal code. I noticed that we hadn't tracked it in our big list here when I was trying to figure out what the next code should be.